tests/providers/test_google.py

```
@@ -1,15 +1,39 @@
+        @property
+        def key_id(self) -> str | None:
+            return None
+
+        def sign(self, message: str | bytes) -> bytes:
+            return b''
+
+    def service_account_credentials(*, scopes: Sequence[str] | None = None) -> service_account.Credentials:
+        return service_account.Credentials(
+            signer=FakeSigner(),
+            service_account_email='service-account@example.com',
+            token_uri='https://oauth2.googleapis.com/token',
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "token_uri"

(S106)